exit/index.js

```diff
@@ -0,0 +1,47 @@
+require('dotenv').config();
+const Web3 = require('web3');
+const MaticPOSClient = require('@maticnetwork/maticjs').MaticPOSClient;
+const Tx = require('ethereumjs-tx').Transaction;
+
+
+const parentProvider = process.env.PARENT_PROVIDER;
+const maticProvider = process.env.MATIC_PROVIDER;
+const senderPrivateKey = process.env.PRIVATE_KEY;
+const rootChainManagerAddress = process.env.ROOT_CHAIN_MANAGER;
```

Comment:
This can be hardcoded to point at the mainnet `RootChainManagerProxy`, rather than pulling it from the `.env`